cmd/cometbft/commands/debug/util.go

```diff
@@ -79,5 +78,5 @@ func dumpProfile(dir, addr, profile string, debug int) error {
 		return fmt.Errorf("failed to read %s profile response body: %w", profile, err)
 	}
 
-	return os.WriteFile(path.Join(dir, fmt.Sprintf("%s.out", profile)), body, os.ModePerm)
+	return os.WriteFile(dir+"/"+profile+".out", body, os.ModePerm)
```

Comment:
```suggestion
	return os.WriteFile(filepath.Join(dir, profile+".out"), body, os.ModePerm)
```
https://pkg.go.dev/path/filepath#Join is better because it's Windows-compatible, which is nice to have imho.